extensions/cacher/README.md

```diff
@@ -0,0 +1,36 @@
+<p align="center">
+  <img src="media/cacher-logo.png" height="128">
+  <h1 align="center">Cacher Extension for Raycast</h1>
+</p>
+
+## About Cacher
+
+[Cacher](https://cacher.io) is a cross-platform, cloud-based app used to curate a snippet library for you and your team. This extension lets you quickly create and insert personal/team snippets using the [Raycast](https://www.raycast.com) launcher.
+
+## Getting Started
+
+1. [Click here](https://app.cacher.io/enter?action=view_api_creds) to view your Cacher API credentials.
+2. Fill in the **API Key** and **API Token** fields for the Cacher Raycast extension.
+3. Once you're signed into Cacher, you can trigger the **Search Code Snippets** and **Create Code Snippet** commands from Raycast.
+
+![Cacher API Credentials](media/api-credentials.png "Cacher API Credentials")
+
+![Cacher Extension Settings](media/raycast-cacher-settings.png "Cacher Extension Settings")
```

Comment:
You can reference this directly from metadata (so you only have them one place, remember to add the optimized to metadata and remove them from media folder)

Reply:
As part of the checklist I see:
- [X] I checked that assets used by the README are placed outside of the metadata folder

Is this not the case anymore?

Reply:
Since the `metadata` screenshots are part of the image carousel users will see in the listing, I'd rather leave the settings out and keep it only in the `README` instructions.